code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -90,7 +90,8 @@
 		var/obj/machinery/atmospherics/pipe/P = A
 		if(P.parent)
 			merge(P.parent)
-		P.parent = src
+		else
```

Comment:
why did you add this `else` statement?

Reply:
`<else>` is not necessary here, just removed one extra assignation.

Reply:
revert this change; that assignment is not redundant

Reply:
merge already have '<P.parent = src>'